features/async-generators.yml

```
@@ -0,0 +1,17 @@
+name: Async generators
+description: Async generator functions (`async function*`) are asynchronous and always yield `Promise` objects.
```

I think "are asynchronous" is redundant to "Async". Do we want to explain generators here as well? Perhaps 

```suggestion
description: Async generator functions (`async function*`) create iterators that return multiple Promises, one after another, on-demand.
```